CMakeLists.txt

```diff
@@ -170,6 +170,10 @@ if("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU" OR
 	set(CMAKE_CXX_FLAGS_MINSIZEREL "-Os")
 	#
 	set(GCC_EXTRA_OPTIONS "")
+        #
+        if( ${BUILD_SHARED_LIBS} )
```

Comment:
This can just be

if(BUILD_SHARED_LIBS)

Also, please make sure the indent is correct.